scripts/o3de/tests/test_repo_properties.py

```diff
@@ -0,0 +1,180 @@
+                         None, None, None, None,
+                         'project1 project2', None, None, [{'project_name':'project1'}, {'project_name':'project2'}],
+                         None, None, None, None,
+                         0),
+            # test remove a project
+            pytest.param(pathlib.Path('F:/repotest'),'Repotest1', 
+                         None, None, None, None,
+                         'project1 project2 project3', 'project2', None, [{'project_name':'project1'}, {'project_name':'project3'}],
+                         None, None, None, None,
+                         0),
+            # test replace a project
+            pytest.param(pathlib.Path('F:/repotest'),'Repotest1', 
+                         None, None, None, None,
+                         'project1 project2', None, 'project3', [{'project_name':'project3'}],
+                         None, None, None, None,
+                         0),
+            # test remove project with a empty parameter
+            pytest.param(pathlib.Path('F:/repotest'),'Repotest1', 
+                         None, None, None, None,
+                         'project1 project2', '', None, [{'project_name':'project1'}, {'project_name':'project2'}],
+                         None, None, None, None,
+                         0),
+            # test add template
+            pytest.param(pathlib.Path('F:/repotest'),'Repotest1', 
+                         None, None, None, None,
+                         None, None, None, None,
+                         'template1 template2 template3', None, None, [{'template_name':'template1'}, {'template_name':'template2'}, 
+                                                                       {'template_name':'template3'}],
+                         0),
+            # test remove multiple templates
+            pytest.param(pathlib.Path('F:/repotest'),'Repotest1', 
+                         None, None, None, None,
+                         None, None, None, None,
+                         'template1 template2 template3', 'template1 template2', None, [{'template_name':'template3'}],
+                         0),
+            # test replace a temlpate
+            pytest.param(pathlib.Path('F:/repotest'),'Repotest1', 
+                         None, None, None, None,
+                         None, None, None, None,
+                         'template1 template2 template3', None, "template48", [{'template_name':'template48'}],
+                         0),
+            # test add all objects
+            pytest.param(pathlib.Path('F:/repotest'),'Repotest1', 
+                         'gem1', None, None, [{'gem_name':'gem1'}],
+                         'project1', None, None, [{'project_name':'project1'}],
+                         'template1 template2', None, None, [{'template_name':'template1'}, {'template_name':'template2'}],
+                         0),
+            # test remove all objects
```

Comment:
This "test remove all object types" not "test remove all objects" right?

Reply:
yes